consent/test/manager_test_helpers.go

```diff
@@ -744,8 +744,8 @@ func ManagerTests(deps Deps, m consent.Manager, clientManager client.Manager, fo
 				})
 			}
 
-			require.EqualError(t, m.RevokeSubjectConsentSession(ctx, "i-do-not-exist"), x.ErrNotFound.Error())
-			require.EqualError(t, m.RevokeSubjectClientConsentSession(ctx, "i-do-not-exist", "i-do-not-exist"), x.ErrNotFound.Error())
+			require.NoError(t, m.RevokeSubjectConsentSession(ctx, "i-do-not-exist"))
```

Comment:
Returning 404 on not found was not documented and is also inconsistent with other apis that return 204 on deletes that touch no rows.